src/Pact/Analyze/Translate.hs

```diff
@@ -434,6 +449,14 @@ translateSchema node = do
     EType _primTy    -> throwError' $ NotConvertibleToSchema $ _aTy node
     EObjectTy schema -> pure schema
 
+translateBinding :: Named Node -> TranslateM (Located Binding)
+translateBinding (Named (Unmunged -> unmunged) node _) = do
+  vid <- genVarId
+  let munged = node ^. aId.tiName.to Munged
+      info = node ^. aId . Pact.tiInfo
+  varType <- translateType node
+  pure $ Located info $ Binding vid unmunged munged varType
```

Comment:
I wonder if we should borrow the `:~` operator [from trifecta](https://hackage.haskell.org/package/trifecta-0.9/docs/Text-Trifecta-Span.html)

Reply:
That seems nice (after this PR)